src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -191,6 +193,16 @@ class ServiceDetails extends React.Component<RouteComponentProps<ServiceId>, Ser
         MessageCenter.add(API.getErrorMsg('Could not fetch Service Details.', error));
         console.log(error);
       });
+    API.getServiceValidations(this.props.match.params.namespace, this.props.match.params.service)
+      .then(response => {
+        this.setState({
+          validations: response['data']
+        });
+      })
+      .catch(error => {
+        MessageCenter.add(API.getErrorMsg('Could not fetch Service Validations.', error));
+        console.log(error);
+      });
```

Comment:
Did you forget remove this console.log ?

Reply:
@aljesusg I did remove them :)